### PR TITLE
tools: killsnoop: support target PID filter

### PR DESCRIPTION
**Pull request by pizhenwei**
Support '-t'/'--tpid' to filter the target PID to avoid message flood, ignore other processes we don't care.

Signed-off-by: zhenwei pi <pizhenwei@bytedance.com>

### CODE REVIEW

**Comment by pizhenwei** [comment]
Hi @devidasjadhav 
Thanks for your suggestion, force pushed a new versoin.

File: man/man8/killsnoop.8 [hunk 1]
```diff
@@ -27,7 +27,10 @@ Print usage message.
 Only print failed kill() syscalls.
 .TP
 \-p PID
-Trace this process ID only (filtered in-kernel).
+Trace this process ID only which is the sender of signal (filtered in-kernel).
+.TP
+\-t PID
+Trace this target process ID only which is the receiver of signal (filtered in-kernel).
```
Comment (yonghong-song) [inline man/man8/killsnoop.8:33]: -t is commonly used for tid. So I would like this option to be -r/--rpid to imply it is for signal receiver.

Comment (pizhenwei) [reply man/man8/killsnoop.8:33]: what about '-T/-tpid'?

Comment (yonghong-song) [reply man/man8/killsnoop.8:33]: '-T/-tpid' is okay to me.

**Comment by pizhenwei** [comment]
@yonghong-song 
Use '-T' instead of '-t', also append another change: auto detect the length of PID/TPID field.
I noticed that the master branch also hits auto test failed...

**Pull request merged**